### DISCUSSION TITLE
Support for roots in next version (Chord module)

### DISCUSSION THREAD

**Discussion by vellebelle** [tonaljs/tonal#287] [Q&A] [answered]
Hi everyone,

I'm working on a project where I need to get the note names for chords with roots. In the docs it says:

Important: currently chord with roots are NOT allowed (will be implemented in next version):

Chord.get("Cmaj7/E"); // => { empty: true }

I would like to ask how far along the next version is and if this is something still being worked on? And if anyone has a workaround - a little hack - that will make it work until, I would be really grateful and happy to learn more about it.

Thanks,

/Morten

**Comment by danigb** [comment, marked as answer]
Hi!

As you can see the development of this library is quite stalled: that is, based mainly on my (very limited) time and the needs of the users.

So, I have no plans to implement this feature, but maybe you can tell me what you need it for and maybe we can think of a solution.

**Comment by vellebelle** [comment]
@danigb Thanks for your answer. I think I found another way to do what I'm trying to do.
